src/Atmos/Model/AtmosModel.jl

```diff
@@ -129,11 +129,7 @@ function AtmosModel{FT}(
     param_set::AbstractParameterSet;
     orientation::O = FlatOrientation(),
     ref_state::RS = HydrostaticState(
-        LinearTemperatureProfile(
-            FT(200),
-            FT(280),
-            FT(grav(param_set)) / FT(cp_d(param_set)),
-        ),
+        DecayingTemperatureProfile(FT(200), FT(60), FT(8 * 10^3)),
```

Comment:
Same as below: Don't you want 
```suggestion
        DecayingTemperatureProfile(FT(290), FT(60), FT(8 * 10^3)),
```
here, i.e., first argument is surface temperature? (The previous parameters had 280 K, but 290 K is closer to Earth's global-mean temperature.

Reply:
The surface temperature, temperature drop, and height scale should probably become CLIMAParameters. The temperatures can be hard-wired parameters. For the height scale, I'd then choose `H = R_d * T_sfc / grav` (which is about 8 km). It would make sense to read the two temperatures from CLIMAParameters, and define the density scale height `H` as a derived parameter.

Reply:
Ok, great. I've added an outer convenience constructor that will do this.